orb/kustomize/orb/components/local-kms/seed.yml

```diff
@@ -0,0 +1,23 @@
+Keys:
+  Asymmetric:
+    Ecc:
+      - Metadata:
+          KeyId: vc-sign
```

Comment:
without the uuid key format, I guess this will limit testing of key rotation.
you can use friendly names if you define key aliases

Reply:
done